DMX.Core/DMX.Core.API/Startup.cs

```diff
@@ -0,0 +1,72 @@
+// --------------------------------------------------------------
+// Copyright (c) Michael Mendelsohn All rights reserved.
+// Liscensed under the MIT License.
+// See License.txt in the project root for licesnse infromation.
+// --------------------------------------------------------------
+
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.HttpsPolicy;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using Microsoft.OpenApi.Models;
+using System;
+using System.Collections.Generic;
```

Comment:
Change the configurations in Visual Studio to honor `System` directives in terms of order before any other directives.

here's how:
1. In Visual Studio, Go to Tools at the top menu
2. Go to Options from the context menu
3. Select Text Editor
4. Select C#
5. Select Advanced
6. Navigate to Checkbox with statement: "Place 'System' directives first when sorting usings"

Here's a screenshot:
![image](https://user-images.githubusercontent.com/1453985/168841532-5ec73d9c-61cd-4a07-8486-c81bb835b9fb.png)

Reply:
Hmm, that box was already checked, but perhaps it wasn't checked at creation time, so it hadn't sorted them since. Several of these are unused, so I will remove them.

Future changes should correctly sort with 'System' first.